src/index.d.ts

```diff
@@ -949,6 +948,18 @@ declare namespace RamdaAdjunct {
          */
         dispatch(functions: Function[]): Function;
 
+        /**
+         * Returns a new list with the item at the position `fromIdx` moved to the position `toIdx`.
+         * If the `toIdx` is out of the `list` range, the item will be placed at the last position
+         * of the `list`. Nagative indices are not supported.
```

Comment:
Would rephrase the `Nagative indices are not supported` to `When negative indices are provided the behavior of the move is unspecified`.